.github/workflows/acceptance.yml

```diff
@@ -51,6 +51,7 @@ jobs:
     permissions:
       contents: read
       id-token: write
+      actions: read # used for DIM metadata
```

Comment:
this seems rare to need -- what is this used for?  usually you can get all the metadata from the actions file on disk without needing special permissions

Reply:
It's specifically for getting the "job" information.
In particular we want the job_id in order to generate deep links to test logs.

If you know any way to get at this without resorting to API, I'm very interested, but nothing I've seen or tried has been able to.

Reply:
I assume you mean this code? https://github.com/getsentry/action-collect-test-data/blob/ac0b9ecb5dc7437c113051e4e6a70012ff02c6dc/lib/set_job_vars.py#L90-L96

this isn't going to work because we use a matrix and so it's just going to give a haphazard job by create time and not necessarily the specific one the test is running in

Reply:
Yes, that. It's designed specifically to work with a matrices, and has worked in all cases that i've thrown at it. See it in action on *this PR*:

0. https://github.com/getsentry/sentry/actions/runs/14473676854/job/40593898542#step:6:116
1. https://github.com/getsentry/sentry/actions/runs/14473676854/job/40593900926#step:6:116
2. https://github.com/getsentry/sentry/actions/runs/14473676854/job/40593901662#step:6:116
3. https://github.com/getsentry/sentry/actions/runs/14473676854/job/40593916799#step:6:116
4. https://github.com/getsentry/sentry/actions/runs/14473676854/job/40593908149#step:6:116
5. and so on

The odds of this randomly happening is ~10^-29.

Reply:
I acknowlege your feeling.
Approval?

Reply:
runner id would be way better than runner name at the very least

Reply:
I'm pretty sure I ruled that out already... but I'll check.

---
checked: yea i did rule out runner_id already.

The runner_id is 217 here. I've also printed out all data that I might correlated it against, but the number 217 only shows up in that one place.

https://github.com/getsentry/action-collect-test-data/actions/runs/14479627338/job/40613598722?pr=25